FlightReservation/components/signUpButtons.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react';
+import {
+    StyleSheet,
+    View,
+    Text,
+    Image,
+    Modal,
+    TouchableOpacity,
+} from 'react-native';
+
+function SignUpButton(props) {
```

Comment:
All this file contains 4 spaces of indentation. But on other components, you are using 2 spaces. You should decide which on to use consistently among all the project.